bundles/org.openhab.binding.sensebox/src/main/resources/ESH-INF/i18n/sensebox_de.properties

```diff
@@ -0,0 +1,51 @@
+binding.sensebox.description = Das senseBox Binding importiert Daten aus dem senseBox Citizen Science-Projekt.
+
+# thing types
+thing-type.sensebox.box.label = senseBox
+thing-type.sensebox.box.description = Ein senseBox Sensor.
+
+# thing type config description
+thing-type.config.sensebox.sensor.senseBoxId.label = senseBox ID
+thing-type.config.sensebox.sensor.senseBoxId.description = Die ID des senseBox Sensors.
+thing-type.config.sensebox.sensor.refreshInterval.label = Abfrageintervall
+thing-type.config.sensebox.sensor.refreshInterval.description = Intervall zur Abfrage der senseBox Daten (in s).
+
+# channel group types
+channel-group-type.sensebox.descriptors.label = senseBox Beschreibung
+channel-group-type.sensebox.descriptors.description = Fasst Daten der senseBox zusammen.
+channel-group-type.sensebox.measurements.label = Messwerte
+channel-group-type.sensebox.measurements.description = Fasst Messwerte zusammen.
+channel-group-type.sensebox.lastReported.label = Letzte Messung
+channel-group-type.sensebox.lastReported.description = Fasst Zeitpunkt der Messwerte zusammen.
+
+# channel types
+channel-type.sensebox.uvIntensity.label = UV-Intensit�t
+channel-type.sensebox.uvIntensity.description = Aktuelle UV-Intensit�t.
+channel-type.sensebox.uvIntensityLastReported.label = Letzte Messung UV-Intensit�t
+channel-type.sensebox.uvIntensityLastReported.description = Zeigt den Zeitpunkt der letzten Messung der UV-Intensit�t an.
+channel-type.sensebox.uvIntensityLastReported.state.pattern = %1$td.%1$tm.%1$tY %1$tH:%1$tM:%1$tS
+channel-type.sensebox.illuminance.label = Beleuchtungsst�rke
+channel-type.sensebox.illuminance.description = Aktuelle Beleuchtungsst�rke.
+channel-type.sensebox.illuminanceLastReported.label = Letzte Messung Beleuchtungsst�rke
+channel-type.sensebox.illuminanceLastReported.description = Zeigt den Zeitpunkt der letzten Messung der Beleuchtungsst�rke an.
+channel-type.sensebox.illuminanceLastReported.state.pattern = %1$td.%1$tm.%1$tY %1$tH:%1$tM:%1$tS
+channel-type.sensebox.pressureLastReported.label = Letzte Messung Luftdrucks
+channel-type.sensebox.pressureLastReported.description = Zeigt den Zeitpunkt der letzten Messung des Luftdrucks an.
+channel-type.sensebox.pressureLastReported.state.pattern = %1$td.%1$tm.%1$tY %1$tH:%1$tM:%1$tS
+channel-type.sensebox.humidityLastReported.label = Letzte Messung Luftfeuchtigkeit
+channel-type.sensebox.humidityLastReported.description = Zeigt den Zeitpunkt der letzten Messung der Luftfeuchtigkeit an.
+channel-type.sensebox.humidityLastReported.state.pattern = %1$td.%1$tm.%1$tY %1$tH:%1$tM:%1$tS
+channel-type.sensebox.temperatureLastReported.label = Letzte Messung Au�entemperatur
+channel-type.sensebox.temperatureLastReported.description = Zeigt den Zeitpunkt der letzten Messung der Au�entemperatur an.
+channel-type.sensebox.temperatureLastReported.state.pattern = %1$td.%1$tm.%1$tY %1$tH:%1$tM:%1$tS
+channel-type.sensebox.particulateMatter2dot5.label = Particulate Matter - PM2,5
```

Comment:
```suggestion
channel-type.sensebox.particulateMatter2dot5.label = Feinstaub - PM2,5
```

Reply:
I think that usually it is also labelled PM2.5 in Germany, maybe keep the dot instead of the comma.